src/test/resources/mockData/configurations/get_configuration_response.json

```diff
@@ -0,0 +1,18 @@
+{
+  "configs":[
+    {
+      "id":"0ee722ff-3dd3-44a2-ae93-8d75297db753",
+      "module":"EXPORT",
+      "configName":"specific_rules",
+      "code":"DATA_EXPORT",
```

Comment:
can we call CODE: "RULES_OVERRIDE"  so that it is readable to know what we are trying to do

Reply:
yes., will rename it